objects/official/scenery_wall/toontowner.scenery_wall.xxwlbr04/object.json

```diff
@@ -0,0 +1,364 @@
+            "srcX": 455,
+            "srcY": 0,
+            "srcWidth": 22,
+            "srcHeight": 37,
+            "x": -1,
+            "y": -32
+        },
+        {
+            "path": "images.png",
+            "srcX": 477,
+            "srcY": 0,
+            "srcWidth": 16,
+            "srcHeight": 40,
+            "x": 16,
+            "y": -24
+        },
+        {
+            "path": "images.png",
+            "srcX": 493,
+            "srcY": 0,
+            "srcWidth": 22,
+            "srcHeight": 37,
+            "x": -20,
+            "y": -33
+        },
+        {
+            "path": "images.png",
+            "srcX": 515,
+            "srcY": 0,
+            "srcWidth": 15,
+            "srcHeight": 39,
+            "x": -31,
+            "y": -24
+        },
+        {
+            "path": "images.png",
+            "srcX": 530,
+            "srcY": 0,
+            "srcWidth": 22,
+            "srcHeight": 37,
+            "x": -1,
+            "y": -32
+        },
+        {
+            "path": "images.png",
+            "srcX": 552,
+            "srcY": 0,
+            "srcWidth": 16,
+            "srcHeight": 40,
+            "x": 16,
+            "y": -24
+        },
+        {
+            "path": "images.png",
+            "srcX": 568,
+            "srcY": 0,
+            "srcWidth": 18,
+            "srcHeight": 37,
+            "x": -16,
+            "y": -33
+        },
+        {
+            "path": "images.png",
+            "srcX": 586,
+            "srcY": 0,
+            "srcWidth": 21,
+            "srcHeight": 39,
+            "x": -31,
+            "y": -24
+        },
+        {
+            "path": "images.png",
+            "srcX": 607,
+            "srcY": 0,
+            "srcWidth": 14,
+            "srcHeight": 37,
+            "x": -1,
+            "y": -32
+        },
+        {
+            "path": "images.png",
+            "srcX": 621,
+            "srcY": 0,
+            "srcWidth": 23,
+            "srcHeight": 41,
+            "x": 9,
+            "y": -25
+        },
+        {
+            "path": "images.png",
+            "srcX": 644,
+            "srcY": 0,
+            "srcWidth": 20,
+            "srcHeight": 37,
+            "x": -18,
+            "y": -33
+        },
+        {
+            "path": "images.png",
+            "srcX": 664,
+            "srcY": 0,
+            "srcWidth": 21,
+            "srcHeight": 38,
+            "x": -31,
+            "y": -23
+        },
+        {
+            "path": "images.png",
+            "srcX": 685,
+            "srcY": 0,
+            "srcWidth": 19,
+            "srcHeight": 37,
+            "x": -1,
+            "y": -32
+        },
+        {
+            "path": "images.png",
+            "srcX": 704,
+            "srcY": 0,
+            "srcWidth": 22,
+            "srcHeight": 39,
+            "x": 10,
+            "y": -23
+        }
+    ],
+    "strings": {
+        "name": {
+            "en-GB": "Wall",
```

Comment:
Brick Wall with Portcullis Door

Reply:
Do you have the translations for other languages for me aswell?